conjur/cli.py

```diff
@@ -93,7 +106,7 @@ def run(self, *args):
         # pylint: disable=line-too-long
         init_subparser = resource_subparsers.add_parser('init',
                                                         help='Initialize the Conjur configuration',
-                                                        description=self.usage('conjur [global options] init [options] [args]'),
+                                                        description=self.command_description('init - Initialize Conjur configuration', 'conjur [global options] init [options] [args]'),
```

Comment:
do not inline in inside this call, move it to a local variable.

Reply:
the string inside?

Reply:
moved string to own var

Reply:
Not exactly what I meant, but it's good enough.